File: backend/Stackoverflow/src/StackOverflowService/StackoverflowService.Application/DTOs/Answers/AnswerDto.cs
using System;

namespace StackoverflowService.Application.DTOs.Answers
{
    public class AnswerDto
    {
        public string Id { get; set; } = default!;
        public string QuestionId { get; set; } = default!;
        public string UserId { get; set; } = default!;
        public string Text { get; set; } = default!;
        public DateTimeOffset CreationDate { get; set; }
        public bool IsFinal { get; set; }
        public bool IsDeleted { get; set; }

        //Aggregations
        public int UpVotes { get; set; }
        public int DownVotes { get; set; }
        public int VoteScore { get; set; }
    }
}

File: frontend/stackoverflow-portal/src/app/core/dto/answer-dto.ts
export interface AnswerDto {
    Id: string;
    QuestionId: string;
    UserId: string;
    Text: string;
    CreationDate: string;
    IsFinal: boolean;
    IsDeleted: boolean;
    User: {
      Id: string;
      Name: string;
      Lastname: string;
      Email: string;
      PhotoBlobName: string | null;
      PhotoContainer: string | null;
    };
    UpVotes: number;
    DownVotes: number;
    VoteScore: number;
    MyVote: number | null;
  }
  